Code/Tools/AssetProcessor/native/AssetManager/PathDependencyManager.cpp

```diff
@@ -370,6 +370,7 @@ namespace AssetProcessor
                 entry.m_dependencySourceGuid = sourceEntry.m_sourceGuid;
                 entry.m_dependencySubID = matchedProduct.m_subID;
                 entry.m_platform = productDependencyDatabaseEntry.m_platform;
+                entry.m_dependencyFlags = AZ::Data::ProductDependencyInfo::CreateFlags(AZ::Data::AssetLoadBehavior::NoLoad);
```

Comment:
I see "m_dependencyFlags" assigned here, but this patch does not seem to use it. Is that correct?

Reply:
Yeah it doesn't use it, this was actually another bug, the 2 different code paths weren't setting the flag to the same value.  Since the other code path was explicitly setting NoLoad, I copied that here (this code path was defaulting to PreLoad).  This is used by the Asset Container system to figure out what it needs to do when loading a dependency.